Subject: [PATCH] that busybox wget is so broken, its terrifying

---
 butch.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/butch/butch.c b/butch/butch.c
index bbfb068..d66e67d 100644
--- a/butch/butch.c
+++ b/butch/butch.c
@@ -403,7 +403,7 @@ int create_script(jobtype ptype, pkgstate* state, pkg* item) {
 			stringptr_fromchar(buf, &tb),
 			SPL(" "),
 			stringptrlist_get(item->data.mirrors, rand() % stringptrlist_getsize(item->data.mirrors)),
-			SPL(" --no-check-certificate"),
+			//SPL(" --no-check-certificate"),
 			NULL);
 		
 	} else if (ptype == JT_BUILD) {
-- 
1.7.3.4

