Subject: [PATCH] busybox wget fails if the resume switch is used on some servers

---
 butch.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/butch/butch.c b/butch/butch.c
index 7b48675..bbfb068 100644
--- a/butch/butch.c
+++ b/butch/butch.c
@@ -401,7 +401,7 @@ int create_script(jobtype ptype, pkgstate* state, pkg* item) {
 			config,
 			SPL("wget -O $C/"),
 			stringptr_fromchar(buf, &tb),
-			SPL(" -c "),
+			SPL(" "),
 			stringptrlist_get(item->data.mirrors, rand() % stringptrlist_getsize(item->data.mirrors)),
 			SPL(" --no-check-certificate"),
 			NULL);
-- 
1.7.3.4

