diff --git a/Makefile b/Makefile
index f027807..c7358dc 100644
--- a/Makefile
+++ b/Makefile
@@ -43,9 +43,14 @@ LDSO_PATHNAME = $(syslibdir)/ld-musl-$(ARCH).so.1
 
 -include config.mak
 
+INSTALL_TARGETS = $(ALL_LIBS:lib/%=$(DESTDIR)$(libdir)/%) $(ALL_INCLUDES:include/%=$(DESTDIR)$(includedir)/%) $(ALL_TOOLS:tools/%=$(DESTDIR)$(bindir)/%)
+ifneq ($(SHARED_LIBS),)
+INSTALL_TARGETS += $(DESTDIR)$(LDSO_PATHNAME)
+endif
+
 all: $(ALL_LIBS) $(ALL_TOOLS)
 
-install: $(ALL_LIBS:lib/%=$(DESTDIR)$(libdir)/%) $(ALL_INCLUDES:include/%=$(DESTDIR)$(includedir)/%) $(ALL_TOOLS:tools/%=$(DESTDIR)$(bindir)/%) $(DESTDIR)$(LDSO_PATHNAME)
+install: $(INSTALL_TARGETS)
 
 clean:
 	rm -f crt/*.o
