From 65db6bf5ea1f27ec2cddd6423868baeff05cc719 Mon Sep 17 00:00:00 2001
From: Rich Felker <dalias@aerifal.cx>
Date: Sat, 17 Mar 2012 21:40:10 -0400
Subject: [PATCH] c++ seems to want some casts in the float representation-access macros

---
 include/math.h |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/math.h b/include/math.h
index 9c06c96..ec4afa7 100644
--- a/include/math.h
+++ b/include/math.h
@@ -41,8 +41,8 @@ int __fpclassify(double);
 int __fpclassifyf(float);
 int __fpclassifyl(long double);
 
-#define __FLOAT_BITS(f) (((union { float __f; __uint32_t __i; }){ (f) }).__i)
-#define __DOUBLE_BITS(f) (((union { double __f; __uint64_t __i; }){ (f) }).__i)
+#define __FLOAT_BITS(f) (((union { float __f; __uint32_t __i; }){ (float)(f) }).__i)
+#define __DOUBLE_BITS(f) (((union { double __f; __uint64_t __i; }){ (double)(f) }).__i)
 
 #define fpclassify(x) ( \
 	sizeof(x) == sizeof(float) ? __fpclassifyf(x) : \
-- 
1.7.3.4

